libraries/js/src/types/general.ts

```diff
@@ -14,11 +14,11 @@ export interface SiwfOptions {
    * Used to make sure the signatures are generated and validated for the correct chain.
    * EIP-712 ChainId is different for different chains
    * Supported Chain Types:
-   * - `Mainnet-Frequency` (Default)
+   * - `Mainnet-Frequency`
    * - `Paseo-Testnet-Frequency`
    * - `Dev`
    */
-  chainType?: ChainType;
+  chainType: ChainType;
```

Comment:
since the inner library needed this, I'm making this a requirement just to ensure it's not going to be using a wrong default. This will break backwards compatibility in some usages